Makefile

```diff
@@ -50,12 +51,14 @@ BUILD_OPTS_DEPLOY?=-mod=vendor "-ldflags=$(BUILDINFO) -w -s"
 check: lint test ## Run linters and tests
 check-windows-appveyor: lint-windows-appveyor test ## Run linters and tests
 
+check-windows: lint test-windows ## Run linters and tests on windows
+
 lint: ## Run linters. Use make install-linters first	
 	${OPTS} golangci-lint run -c .golangci.yml ./...
 	# The govet version in golangci-lint is out of date and has spurious warnings, run it separately
```

Comment:
> I still can't reproduce it. Have you got hmm diffutils installed on windows?
If I remove this comment then everything works well and all the tests pass.

Reply:
removed comment in line number 58.

Reply:
Looks like you forgot to push the commit.